docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/DockerCommandLocations.java

```diff
@@ -1,47 +1,75 @@
 /*
  * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
```

Comment:
All files should have the Apache licence. I'll fix the ones that had it removed.